client/my-sites/people/people-list-section-header/index.jsx

```diff
@@ -42,6 +43,10 @@ class PeopleListSectionHeader extends Component {
 			return '/people/email-followers/' + siteSlug;
 		}
 
+		if ( this.isViewersTab() ) {
+			return `/people/new/${ siteSlug }?role=viewer`;
```

Comment:
The idea is to pre-select the viewer role in the new user form. Let me know if there is already, or a better way, of doing that.